src/main/java/org/broadinstitute/hellbender/engine/FeatureManager.java

```diff
@@ -80,7 +80,7 @@
      * The tool instance containing the FeatureInput argument values that will form the basis of our
      * pool of FeatureDataSources
      */
-    private final CommandLineProgram toolInstance;
+    private transient final CommandLineProgram toolInstance;
```

Comment:
Possibly. The tests work with this change (this includes tests that load features that aren't in this branch), so it's useful for some cases.